src/app/modules/gb-analysis-module/cross-table-components/gb-cross-table/gb-cross-table.component.ts

```diff
@@ -23,6 +24,8 @@ import {FormatHelper} from '../../../../utilities/format-helper';
 })
 export class GbCrossTableComponent implements OnInit {
 
+  AxisType = AxisType;
```

Comment:
Why is this needed?

Reply:
Otherwise you cannot use the type in the html template.